persistence-jdbc/core/src/main/scala/com/lightbend/lagom/internal/persistence/jdbc/SlickProvider.scala

```diff
@@ -167,7 +167,12 @@ private[lagom] class SlickProvider(system: ActorSystem)(implicit ec: ExecutionCo
     // Calling MTable.getTables without parameters fails on MySQL
     // See https://github.com/lagom/lagom/issues/446
     // and https://github.com/slick/slick/issues/1692
-    MTable.getTables(None, currentSchema, Option("%"), None)
+    profile match {
+      case _: MySQLProfile =>
+        MTable.getTables(currentSchema, None, Option("%"), None)
+      case _ =>
+        MTable.getTables(None, currentSchema, Option("%"), None)
+    }
```

Comment:
If this is only happening with drive 6.0.0+, that change will break for versions previous to 6.0.0. Or am I missing something?